lib/fizzbuzz.rb

```diff
@@ -1,9 +1,17 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+  
+	  case 
```

Comment:
It can be done with a case statement, but it feels like it's a little overkill for what's being done.